app/dashboard/_components/InterviewsTable/NetworkVisualization.tsx

```diff
@@ -0,0 +1,172 @@
+import type { Codebook, NcNetwork } from '@codaco/shared-consts';
+import { useEffect, useState } from 'react';
+import { getCSSVariableAsString } from '~/lib/ui/utils/CSSVariables';
+
+const NODE_COLOR_FALLBACK = 'hsl(342, 76.9%, 50.8%)'; // neon coral
+const NODE_COLOR_DARK_FALLBACK = 'hsl(342, 76.9%, 45.8%)'; // neon coral dark
+const EDGE_COLOR_FALLBACK = 'hsl(46, 100%, 47%)'; // mustard
+const EDGE_COLOR_DARK_FALLBACK = 'hsl(46, 100%, 42%)'; // mustard dark
```

Comment:
I don't think these should be needed.

Reply:
resolved [bff0149](https://github.com/complexdatacollective/Fresco/pull/363/commits/bff01494fc5525c7e24a24bb10126b798b1b73e6)